src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+		<p><b>* NOTE</b>: By default, once the <code className="small-code-name">enableViewSecurityId</code> property in the configuration.json for an Add-in is set to True, a View “Add-in name” add-in identifier is created that must be enabled for users to be able to view the Add-in. More granular control needs to be set within the Add-in code for any custom identifiers added to the configuration.json.</p>
+		<h2>IP Rate LimitsPermalink</h2>
+		<ul>
+			<li>
+				Added new rate limits for API requests for endpoints that do not require authentication. See <a href="https://docs.google.com/document/d/1sUKaOKjVT23qvaCITMseIMqGUhQzlr7xDhq1EjrseYE/edit#heading=h.ygvbrm3xiv4u">this announcement</a> for more information.
+			</li>
+		</ul>
+		<h2>Updates</h2>
+		<h3>DatabaseExists</h3>
+		<ul>
+			<li>
+				Removed the DatabaseExists API that was originally used for database registration purposes. If the database name exists, <code className="small-code-sample">CreateDatabase</code> throws a <code className="small-code-sample">RegistrationException</code>.
+			</li>
+		</ul>
+		<h3>RouteSearch</h3>
+		<ul>
+			<li>
+				Removed the <code className="small-code-sample">RouteSearchGroups</code> property and replaced it with the <code className="small-code-sample">ServiceGroups</code> property.
+			</li>
+		</ul>
+		<h3>PropertySelector</h3>
+		<ul>
+			<li>
+				Added the optional  <code className="small-code-sample">PropertySelector</code> parameter that is used with the <code className="small-code-sample">Get</code> and <code className="small-code-sample">GetFeed</code> methods to include or exclude specific properties for entity types requested. Detailed information can be found on the <a href="https://geotab.github.io/sdk/software/guides/concepts/#propertyselector-beta">Geotab Developers</a> site.
+			</li>
+		</ul>
+		<h3>DriverWhiteListContent / DriverAuthListContent</h3>
+		<ul>
+			<li>
+				Removed the <code className="small-code-sample">DriverWhiteListContent</code> object and replaced it with the <code className="small-code-sample">DriverAuthListContent</code> object.
+			</li>
+		</ul>
+		<h3>Group</h3>
+		<ul>
+			<li>
+				Improved system group objects to be returned with an English Name field.
+			</li>
+		</ul>
+		<h3>DeviceShareStatus</h3>
+		<ul>
+			<li>
+				Added <code className="small-code-sample">RequestPending</code> as an acceptable value of <code className="small-code-sample">DeviceShareStatus</code>. This value indicates when an outgoing device share request has been created by a user on the source database, and is waiting for confirmation it has been created successfully.
+			</li>
+		</ul>
+		<h3>SecurityIdentifier</h3>
+		<ul>
+			<li>
+				Added the following security identifiers:
+				<ul>
+					<li>
+						<code className="small-code-sample">ResetPassword</code>
```

Comment:
Content inside of code tags should be surrounded by curly brackets & ticks { `` } for all of the code tags in this ul tag